api/http_client.go

```diff
@@ -147,11 +147,20 @@ func (c HTTPClient) TryGetLock(ctx context.Context, id string) error {
 	return nil
 }
 
-func (c HTTPClient) AddIP(ctx context.Context, ip string, checks ...models.Healthcheck) (IP, error) {
+type AddIPParams struct {
+	IP                  string               `json:"ip"`
```

Comment:
The IP argument being mandatory, I would prefer the following signature for this function:

```go
 func (c HTTPClient) AddIP(ctx context.Context, ip string, params AddIPParams) (IP, error) { 
```
What do you think?

Reply:
I agree